### ISSUE TITLE
Strobogromatic Number (CP) in Java

### ISSUE THREAD

**Issue by Rajkanwars15** [sobhanbera/algorithms#222]
#### This issue is about [Algo/DS Name/Question Name](link to resource for the Algo/DS/Question)

-   [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate

-   [x] Issue Title is in form `[Algo/DS Name] in [Language]` (ex:- BubbleSort in Java)

-   [x] I will send my PR only after I'm assigned to this issue by any maintainer

-   [x] Assign this issue to me (I want to work on this)

-   [x] I ensure that I am not already assigned to 2 or more issues

https://leetcode.com/problems/strobogrammatic-number/description/
Strobogrammatic Number is a number whose numeral is rotationally symmetric so that it appears the same when rotated 180 degrees. In other words, Strobogrammatic Number appears the same right-side up and upside down.

0 after 180° rotation : (0 → 0)
1 after 180° rotation : (1 → 1)
8 after 180° rotation : (8 → 8)
6 after 180° rotation : (6 → 9)
9 after 180° rotation : (9 → 6)

**Comment by FardeenCODEIIEST** [comment]
Hey @Rajkanwars15 , can you please assign me this task.

**Comment by sobhanbera** [comment]
Hello @Rajkanwars15,
Thanks for the issue, I've assigned the issue to you. You can now start working on it!

**Comment by sobhanbera** [comment]
Hey @FardeenCODEIIEST,
Issues are FCFS basis. There are lot more algos to implement. You can open a new issue of your own as well and work on it.

**Comment by Rajkanwars15** [comment]
Hey @sobhanbera I've made the PR. If you can please review and accept

**Comment by Prathamesh-510** [comment]
Hey @sobhanbera I want to contribute in this task so please assign this task
